cmd/http/headers.go

```diff
@@ -72,10 +72,10 @@ const (
 	AmzCopySourceVersionID        = "X-Amz-Copy-Source-Version-Id"
 	AmzCopySourceRange            = "X-Amz-Copy-Source-Range"
 	AmzMetadataDirective          = "X-Amz-Metadata-Directive"
-	AmzObjectLockMode             = "X-Amz-Object-Lock-Mode"
-	AmzObjectLockRetainUntilDate  = "X-Amz-Object-Lock-Retain-Until-Date"
-	AmzObjectLockLegalHold        = "X-Amz-Object-Lock-Legal-Hold"
-	AmzObjectLockBypassGovernance = "X-Amz-Bypass-Governance-Retention"
+	AmzObjectLockMode             = "x-amz-object-lock-mode"
+	AmzObjectLockRetainUntilDate  = "x-amz-object-lock-retain-until-date"
+	AmzObjectLockLegalHold        = "x-amz-object-lock-legal-hold"
+	AmzObjectLockBypassGovernance = "x-amz-bypass-governance-retention"
```

Comment:
not required to lower case the header - http.Header's Get() method is case insensitive.

Reply:
done